File: leetcode-questions/java/src/test/java/com/leetcode/questions/twosum/TwoSumTest.java
package com.leetcode.questions.twosum;

import com.leetcode.questions.leetcode.editor.en.TwoSum;
import com.leetcode.questions.support.GeneratedLeetCodeTestHarness;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.nio.file.Files;
import java.nio.file.Path;

public class TwoSumTest {

    private static final Path CASES_PATH = Path.of("generated-test-cases", "two-sum.json");

    public TwoSumTest() {

    }

    @Test
    public void testCases() throws Exception {
        GeneratedLeetCodeTestHarness.run(TwoSum.Solution.class, "twoSum", Files.readString(CASES_PATH));
    }
}

File: leetcode-questions/csharp/src/test/csharp/com/leetcode/questions/twosum/TwoSumTest.cs
using Xunit;
using System.Collections.Generic;
using com.leetcode.questions.leetcode.editor.en.twosum;
using com.leetcode.questions.support;

namespace com.leetcode.questions.twosum;

public class TwoSumTest {

    private const string CasesFileName = "two-sum.json";

    public TwoSumTest() {

    }

    [Fact]
    public void TestCases() {
        GeneratedLeetCodeTestHarness.Run(typeof(Solution), "TwoSum", GeneratedLeetCodeTestHarness.ReadCasesFile(CasesFileName));
    }
}
